### ISSUE TITLE
Not working when project has no Additional Include Directories

### ISSUE THREAD

**Issue by yoko1998** [Caphyon/clang-power-tools#295]
Fresh issue after last update, clang stopped working anymore because now empty IncludeDirectories are not supported.
Putting any dummy directory into IncludeDirectories helps.
VS2017

```

Start Clang Compile

PROJECT: C:\Users\aaa\source\repos\playpen\overlapped\overlapped.vcxproj
Get-ProjectStdafxDir : Cannot bind argument to parameter 'additionalIncludeDirectories' because it is null.
At C:\Users\aaa\AppData\Local\Microsoft\VisualStudio\15.0_08aed11f\Extensions\u3g1t4ew.v1y\clang-build.ps1:2132 
char:69
+ ...           -additionalIncludeDirectories $additionalIncludeDirectories
+                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Get-ProjectStdafxDir], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Get-ProjectStdafxDir
 
```

**Comment by hero101111** [comment]
Hi,

You're right, our latest version (`3.2`) introduced this bug. We've already identified and fixed the issue.

A bugfix release will be released as soon as possible.
Until then you can manually replace `clang-build.ps1` with the latest from github.

Sorry for the inconvenience.

Regards,
Gabriel

---

To manually replace `clang-build.ps1` : 
- Search for `clang-build.ps1` in `%LOCALAPPDATA%`
- Replace it with the github version
- Restart Visual Studio

**Comment by yoko1998** [comment]
Thank you for a swift response.

**Comment by Enache-Ionut** [comment]
Hi,

This issue was fixed and it was implemented in [version 3.2.1](https://marketplace.visualstudio.com/items?itemName=vs-publisher-690586.ClangPowerTools) just released.
Thank you for your feedback! 😃
